packages/cycle-scripts-one-fits-all/template/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "compileOnSave": false,
     "compilerOptions":
     {
-        "module": "ES6",
```

Comment:
Weird. [This](https://www.typescriptlang.org/docs/handbook/compiler-options.html) lists all possible values, and they are all upper case